public/app/plugins/datasource/grafana-azure-monitor-datasource/components/ArgQueryEditor/ArgQueryEditor.tsx

```diff
@@ -22,11 +23,34 @@ const ArgQueryEditor: React.FC<LogsQueryEditorProps> = ({
   onChange,
   setError,
 }) => {
+  const [subscriptions, setSubscriptions] = useState<AzureMonitorOption[]>([]);
+
+  useEffect(() => {
+    datasource.azureMonitorDatasource
+      .getSubscriptions()
+      .then((results) => {
+        const newSubscriptions = results.map((v) => ({ label: v.text, value: v.value, description: v.value }));
```

Comment:
very nitpicky of me but maybe instead of newSubscriptions it's like fetchedSubscriptions or something